src/core/agent/services/signifyNotificationService.ts

```diff
@@ -218,6 +231,50 @@ class SignifyNotificationService extends AgentService {
     });
     return notificationRecord;
   }
+  async onSignifyOperationStateChanged(
+    callback: ({ record, recordType }: {record: IdentifierShortDetails, recordType: OperationPendingRecordType}) => void
+  ) {
+    // eslint-disable-next-line no-constant-condition
+    while (true) {
+      const pendingOperations =
+        await this.operationPendingStorage.getAll();
+      if (pendingOperations.length > 0) {
+        for (const pendingOperation of pendingOperations) {
+          const operation = await  this.signifyClient.operations().get(pendingOperation.id);
+          if (operation.done) {
+            switch (pendingOperation.recordType) {
+            case OperationPendingRecordType.IDENTIFIER: {
+              const aid = await this.identifierStorage.getIdentifierMetadata(pendingOperation.recordId);
+              await this.identifierStorage.updateIdentifierMetadata(aid.id, {
+                isPending: false,
+              });
+              callback({
+                recordType: pendingOperation.recordType,
+                record: {
```

Comment:
I think we can pass something more like:
```
opType: pendingOperation.recordType,
oid: aid.id
```

`oid` is the term used in KERIA. In the UI, we should be able to update the current list of identifiers by ID to `isPending: false`, right? This would be cleaner I think